spec/lib/reports/report_renderer_spec.rb

```diff
@@ -35,3 +35,129 @@
     end
   end
 end
+
+# --- metadata headers coverage ---
+RSpec.describe Reporting::ReportRenderer do
```

Comment:
We prefer to have only one RSpec.describe block per file and then nested `describe` blocks for different cases.